playbooks/aws/openshift-master/config.yml

```diff
@@ -10,6 +10,7 @@
 - name: "Gather facts for minions in {{ oo_env }}"
   hosts: "tag_env-host-type_{{ oo_env }}-openshift-minion"
   connection: ssh
+  accelerate: false
```

Comment:
Is this still needed? As far as I can tell it would only be used when running from a RHEL 6 host that doesn't support ControlPersist.

I could see it being a configurable item though, so that if the user is running from a RHEL6 host and cannot connect to the default accelerate port, then they could disable it.